design/Logo/Logo_vertical.svg

```diff
@@ -0,0 +1,19 @@
+<path d="M881.377 307.797V272.885L900.673 273.139C904.059 273.139 907.317 272.589 910.449 271.488C913.58 270.388 916.458 268.865 919.082 266.918C921.79 264.971 924.16 262.644 926.191 259.936C928.222 257.312 929.873 254.434 931.142 251.303C927.334 252.911 923.44 254.519 919.463 256.127C915.569 257.735 911.634 258.539 907.656 258.539C899.277 258.539 891.406 257.058 884.043 254.096C876.679 251.133 870.205 246.944 864.619 241.527C859.117 236.026 854.759 229.382 851.543 221.596C848.411 213.725 846.845 204.923 846.845 195.189V120.033H881.377V195.189C881.377 200.014 882.054 204.245 883.408 207.885C884.847 211.439 886.751 214.444 889.121 216.898C891.491 219.268 894.241 221.046 897.373 222.23C900.589 223.415 904.017 224.008 907.656 224.008C911.211 224.008 914.554 223.204 917.685 221.596C920.901 219.903 923.694 217.702 926.064 214.994C928.434 212.286 930.296 209.239 931.65 205.854C933.004 202.383 933.681 198.829 933.681 195.189V120.033H968.593V238.861C968.509 248.425 966.647 257.396 963.007 265.775C959.368 274.154 954.375 281.475 948.027 287.738C941.764 294.001 934.443 298.952 926.064 302.592C917.685 306.231 908.714 308.051 899.15 308.051L881.377 307.797Z" fill="white"/>
+</g>
+<defs>
+<filter id="filter0_d" x="85.8669" y="65.9512" width="886.727" height="696.785" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB">
+<feFlood flood-opacity="0" result="BackgroundImageFix"/>
+<feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/>
+<feOffset dy="4"/>
+<feGaussianBlur stdDeviation="2"/>
+<feColorMatrix type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.25 0"/>
+<feBlend mode="normal" in2="BackgroundImageFix" result="effect1_dropShadow"/>
+<feBlend mode="normal" in="SourceGraphic" in2="effect1_dropShadow" result="shape"/>
+</filter>
+</defs>
+</svg>
```

Comment:
This is a GENERAL comment for all the stuff in the design folder. 

You already have them all in frontend, so I think we do not need them in the design folder anymore (or we could have the design folder in the frontend folder).